sfdx-project.json

```diff
@@ -4,8 +4,8 @@
             "default": true,
             "package": "apex-rollup",
             "path": "rollup",
-            "versionNumber": "1.2.40.0",
-            "versionDescription": "Updated annotations to proper casing, fixed requeueing bug",
+            "versionNumber": "1.2.41.0",
```

Comment:
Just curious, what's your approach for deciding to increment the patch version vs the minor version?

Reply:
I probably should have upgraded the minor number when releasing the rollup plugins. It's a good question - despite the conversations we've had about it, I have just been incrementing the patch version. I think once I add support for #152, I'll increment the minor number as well to reflect that.